thirdparty/cnpy/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.0 FATAL_ERROR)
+if(COMMAND cmake_policy)
+	cmake_policy(SET CMP0003 NEW)
+endif(COMMAND cmake_policy)
+
+project(CNPY)
+
+set(BUILD_SHARED_LIBS OFF)  
+set(TARGET_NAME "cnpy")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
we already have global `CMAKE_CXX_STANDARD` set in samples and developer package

Reply:
It was deleted.